sample_consensus/include/pcl/sample_consensus/impl/sac_model_circle.hpp

```diff
@@ -100,6 +100,26 @@ pcl::SampleConsensusModelCircle2D<PointT>::computeModelCoefficients (const std::
   return (true);
 }
 
+#ifdef __AVX__
```

Comment:
Maybe a unified macro for `__AVX__ && __AVX__2`? This function is AVX only, but the usage is when AVX2 is also present. What do you think? Same with SSE

Reply:
This function could be used in SIMD implementations of `selectWithinDistance()` and `getDistancesToModel()` in the future. I think those implementations would also be AVX only.